tests/config-tests.js

```diff
@@ -12,17 +12,18 @@ function formatErrors (errors) {
     return errors.map(item => `${item.instancePath}: ${item.message}`).join(', ')
 }
 
+// Test the latest 2 versions of each platform
 const v2configs = platforms.map((plat) => {
```

Comment:
These variable names don't match what we're testing. Should we rename or test all variants?